spec/README.md

```diff
@@ -269,6 +269,45 @@ To run the test suite against such a server, run:
 
     MONGODB_URI="mongodb://localhost:27017/?authMechanism=MONGODB-X509&tls=true&tlsCAFile=spec/support/certificates/ca.crt&tlsCertificateKeyFile=spec/support/certificates/client-x509.pem" rake
 
+## Field-Level Encryption
+
+Install libmongocrypt on your machine:
+
+Option 1: Download a pre-built binary
+- Download a tarball of all libmongocrypt variations from this link:
+    https://s3.amazonaws.com/mciuploads/libmongocrypt/all/master/latest/libmongocrypt-all.tar.gz
+- Unzip the file you downloaded. You will see a list of folders, each
+    corresponding to an operating system. Find the folder that matches
+    your operating system and open it.
+- Inside that folder, open the folder called "nocrypto." In either the
+    lib or lb64 folder, you will find the libmongocrypt.so or
+    libmongocrypt.dylib or libmongocrypt.dll file, depending on your OS.
+- Move that file to wherever you want to keep it on your machine.
+
+Option 2: Build from source
+- To build libmongocrypt from source, follow the instructions in the README on the libmongocrypt GitHub repo: https://github.com/mongodb/libmongocrypt
+
+In one terminal, launch MongoDB:
+
+NOTE: You must be running MongoDB 4.2 or higher. All auto-encryption features
+will require an enterprise build of MongoDB, but you can still run
+explicit encryption tests without enterprise.
+
+Download different versions of MongoDB here: https://www.mongodb.com/download-center/community
```

Comment:
This is the first set of tests that requires an enterprise build. Should I go more in depth about downloading and setting up a different MDB version?

Reply:
Should the link be https://www.mongodb.com/download-center/enterprise for the enterprise version?